vendor/engines/sanger_sequencing/spec/javascripts/sanger_sequencing/print_warning_spec.js.coffee

```diff
@@ -0,0 +1,19 @@
+#= require sanger_sequencing/print_warning
+#= require helpers/jasmine-jquery
+
+describe "Print Warning", ->
+
+  it "does nothing without the right thing in place", ->
+    warning = new PrintWarning
+    spyOn(warning, "listen")
+    expect(warning.shouldWarn()).toBeFalsy()
+    warning.initListener()
+    expect(warning.listen).not.toHaveBeenCalled()
+
+  it "warns when the right elements are in place", ->
+    warning = new PrintWarning
+    spyOn(warning, "listen")
+    setFixtures($("<h2>").addClass("js--print-warning"))
+    expect(warning.shouldWarn()).toBeTruthy()
+    warning.initListener()
+    expect(warning.listen).toHaveBeenCalled()
```

Comment:
Couldn't find an easy way to test triggering the listener, but there's no logic there that isn't already covered by the browser code, presumably.